python/paddle/fluid/layers/nn.py

```diff
@@ -3292,3 +3293,98 @@ def autoincreased_step_counter(counter_name=None, begin=1, step=1):
         counter.stop_gradient = True
 
     return counter
+
+
+def lod_reset(x, y, target_lod=None):
```

Comment:
```python
def lod_reset(x, y=None, target_lod=None):
```

Reply:
y == None and target_lod == None? Seems that we prefer target lod from y.